_format/1.1/index.md

```diff
@@ -1839,3 +1877,572 @@ An error object **MAY** have the following members:
 [error details]: #errors
 [member names]: #document-member-names
 [pagination]: #fetching-pagination
+
+## <a href="#operations" id="operations" class="headerlink"></a> Operations
+
+A server **MAY** support usage of the `operations` array to complete one or more
+fetch or mutation operations in a single request.
+
+All HTTP requests that include an `operations` array **MUST** be issued with
+`PATCH`.
+
+For the sake of clarity, requests that include operations are called
+"operations requests" while requests that perform a single fetch or mutation are
+called "singular requests".
+
+### <a href="#operation-objects" id="operation-objects" class="headerlink"></a> Operation Objects
+
+An operation object **MAY** contain any of the following members:
+
+* `op`: an operation code, expressed as a string, that indicates the type of
+  operation to perform. Valid values include:
+  - `"get"`: fetches a resource, resource identity, collection of resources, or
+     collection of resource identitities
+  - `"add"`: creates a new resource or relationship
+  - `"update"`: updates a resource or relationship
+  - `"remove"`: removes a resource or relationship
+
+* `data`: the operation's "primary data".
+
+* `included`: an array of [resource
+  objects](#document-resource-objects) that are
+  related to the operation's primary data and/or each other
+  ("included resources").
+
+* `ref`: the resource or resource collection that is the target of the
+  operation. This member serves to disambiguate the target at a particular
+  endpoint if that endpoint represents multiple resources. `ref` **MUST** be
+  expressed as an object that **MAY** contain any of the following members:
+  - `type`: resource `type`
+  - `id` or `lid`: resource identifier or local identifier
```

Comment:
It _may_ be useful to provide an example when you might include a `lid` member on a `ref`. Personally, I'm struggling to see a use case for it. From what I can tell the only time I need a `lid` is when I am creating a new resource and need to reference that resource somewhere else in the same request. The only place I can imagine **needing** to reference it somewhere else would be as a relationship to another resource in a resource identifier object. Other possible scenarios this allows:

Add followed by a get.
This is not necessary since `a` is returned by the first operation
```
0: add a
1: get a
```

Add followed by an update or remove
I can't see a purpose for this.
```
0: add a
1: update a
2: remove a
```